src/cms/adobe-stock-manage.md

```diff
@@ -0,0 +1,210 @@
+
+    <!--{% if "Default.CE Only" contains site.edition %}-->
+    - If you have WYSIWYG mode disabled, click **Insert Image**.
+
+    - If you have WYSIWYG mode enabled, click **Show/Hide Editor** and then click **Insert Image**.
+    <!--{% endif %}-->
+    <!--{% if "Default.EE-B2B" contains site.edition %}-->
+    - If you have [Page Builder enabled]({% link cms/page-builder-setup.md %}), expand the **Media** panel and drag an **Image** placeholder to the target container. Then click <span class="btn">Select from Gallery</span>.
+
+      ![]({% link images/images-ee/page-builder-media-image-drag.png %}){: .zoom}
+      _Drag Image to Stage_
+
+    - If you have the [WYSIWYG editor enabled]({% link cms/editor-configure.md %}), click **Show/Hide Editor** and then click **Insert Image**.
+    <!--{% endif %}-->
+
+1. Click <span class="btn">Search Adobe Stock</span>.
+
+{:.procedure}
+To search Adobe Stock assets and add a stock image to a category:
+
+1. On the _Admin_ sidebar, go to **Catalog** > **Categories**.
+
+1. Click **Add Root Category** or **Add Subcategory**.
+
+   If you want to add the image to an existing category, click the category name in the list on the left..
+
+1. Expand the **Content** section, and under **Category Image** click **Select from Gallery**.
+
+1. Click <span class="btn">Search Adobe Stock</span>.
+
+{:.procedure}
+To search Adobe Stock assets and add a stock image from the WYSIWYG Editor:
+
+1. Click **Show/Hide Editor**.
+
+1. Click **Insert Image**.
+
+1. Click <span class="btn">Search Adobe Stock</span>.
+
+    ![Adobe Stock search results]({% link images/images/adobe-stock/search-grid.png %}){: .zoom}
+    _Adobe Stock search results_
+
+## Filter and Search for Adobe Stock Assets
+
+The [Adobe Stock Search Grid][access-search] provides querying and filtering functionality to help you find the perfect image for your Magento stores.
+
+By default, the search results shown are from an Adobe Stock curated gallery of a few hundred results. As soon as you apply your own keyword search, you will be
+searching the millions of assets available via Adobe Stock.
+
+### Search for Adobe Stock Assets by Keywords
+
+1. [Access the Adobe Stock Search grid][access-search].
+
+1. Enter your keyword search into the **Search by keyword** input field in the top-left and click the magnifying glass or press **Enter**.
+
+    ![Adobe Stock Search Results for the "mango" keyword]({% link images/images/adobe-stock/mango.png %}){: .zoom}
+    _Adobe Stock search results - "mango" keyword_
+
+### Filter Adobe Stock Assets
+
+1. [Run a keyword search for Adobe Stock assets][search-by-keywords].
+
+1. Click <span class="btn">Filters</span>.
+
+    There are several filters available to refine your search results:
+
+    |Filter|Description|
+    |---|---|
+    |Subcategory|Filter for images that are **Photos** or **Illustrations**|
+    |Orientation|Filter for images by size, shape and aspect|
```

Comment:
all of this looks bold, is it a glitch?   i see the ** are closed.

Reply:
it's a github display thing